src/views/OpenOrders.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>10 of 26 orders</ion-title>
+        <ion-buttons  slot="end">
+        <ion-menu-button>
+          <ion-icon :icon="options" />
+        </ion-menu-button>
+        </ion-buttons>
+      </ion-toolbar>
+    </ion-header>
+    
+    <ion-content :fullscreen="true">
+      <ion-searchbar /> 
+
+      <div class="filters">
+        <ion-row class="filter-scroll">
+          <ion-item lines="none">
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Same day</h5>
+              <p>37 orders, 40 items</p>
+            </ion-label>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Next day</h5>
+              <p>37 orders, 40 items</p>
+            </ion-label>
+          </ion-item>
+          <ion-item lines="none">
+            <ion-checkbox slot="start"/>
+            <ion-label>
+              <h5>Loyalty</h5>
+              <p>37 orders, 40 items</p>
+            </ion-label>
+          </ion-item>
+        </ion-row>
+      </div> 
+
+      <ion-button class="desktop-print" fill="outline" @click="assignPickers">Print Picksheet</ion-button>
```

Comment:
change class name to desktop-only, make it a generic utility class and set it globally

Reply:
Done